hpi-schul-cloud/nuxt-client
# fix logo loading

# Short Description

QA note:
Change tested, no issues found. More details see comments in ticket https://ticketsystem.dbildungscloud.de/browse/BC-3565




## Links to Ticket and related Pull-Requests
https://ticketsystem.dbildungscloud.de/browse/BC-3565


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
